api/serializers/summary_serializer.py

```diff
@@ -0,0 +1,12 @@
+from rest_framework import serializers
+
+# create a serializer
+class SummarySerializer(serializers.Serializer):
+    # initialize fields
+    model = serializers.CharField()
+    max_tokens = serializers.IntegerField()
+    messages = serializers.ListField(
+        child=serializers.DictField(
```

Comment:
What is a DictFiled? I also find this interesting how nest fields with in others!

Reply:
The DictField is used to represent each Item in the list as a dictionary. Such as in the messages list is a dictionary with string keys and values.